src/createServer.js

```diff
@@ -1,8 +1,57 @@
+/* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const PUBLIC_DIR = 'public';
+  const PUBLIC_DIR_PATH = path.resolve(__dirname, `../${PUBLIC_DIR}`);
+  const FILE_PATH = '/file/';
+  const INDEX = 'index.html';
+
+  return http.createServer(async(req, res) => {
+    res.setHeader('Content-Type', 'text/plain');
+
+    const { pathname } = new URL(
+      req.url,
+      `http://${req.headers.host}`
+    );
+
+    if (!pathname.startsWith(FILE_PATH)) {
+      res.statusCode = 200;
```

Comment:
```suggestion
      res.statusCode = 400;
```

Reply:
But the test requires status code 200.

it('should return hint message for routes not starting with /file/', async() => {
      const response = await axios.get(`${HOST}/file`);

      expect(response.status).toBe(200);
      expect(response.headers['content-type']).toBe('text/plain');
      expect(response.data.length).toBeGreaterThan(0);
});